### ISSUE TITLE
False positive on ControlStructures.RequireTernaryOperator.TernaryOperatorNotUsed

### ISSUE THREAD

**Issue by Dropa** [slevomat/coding-standard#1378]
This triggers error, and seems like a bug as there's nothing to assing.
```php
    if (empty($value) || $value === 'new') {
```

**Comment by kukulich** [comment]
Please post the complete code. The condition is not complete.

**Comment by Dropa** [comment]
Ah, you're right, sorry. After cleaning all trash from the code I can see it kinda gets down to this, and it as a matter of fact is assignment, though slightly long one to use with ternary.

```php
  public static function getNodeOrCreate($entity_id) {
    if (empty($entity_id) || $entity_id == 'new') {
      $node = Node::create([
        'type' => 'post',
      ]);
    }
    else {
      $node = Node::load($entity_id);
    }
    return $node;
  }
  ```
